test/integration/lib/mascara-first-time.js

```diff
@@ -57,8 +57,14 @@ async function runFirstTimeUsageTest (assert, done) {
 
   await timeout(1000)
 
+  // Skip things before Privacy:
```

Comment:
I think that Privacy notice should go first and that unless we change this otherwise shouldnt show notices to new users